.github/workflows/ci.yml

```
@@ -0,0 +1,71 @@
+name: CI
+on: [push]
+jobs:
+  python35-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Apt install
+        run: cat Aptfile | sudo xargs apt-get install
+
+      - name: Python setup
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.5"
+
+      - id: cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-
+          restore-keys: |
+            ${{ runner.os }}-pip-
+
+      - name: Poetry install
+        run: pip install Poetry
```

Poetry is dropping py3.5 in the next version, so we should pin the version here
```suggestion
        run: pip install  -Iv Poetry== 1.1.4
```